platforms/dotnet/versions/stretch/versionsToBuild.txt

```diff
@@ -60,3 +60,4 @@
 7.0.201, fc9d224bf1d3600e878991fc1e8d3b1a0f21c7a8aac7b3cae0e4925ad33172cc12f56210eabfd66cfedd5f70f85918b889673401172b3999cecbeb8f2fe58863,
 7.0.202, f415a8e6c078421759a963aa0b232c092ecf2f0a8e014ba72092390aac792ed35e8f3c822b2ce91ed636cdee9342bba2b89fb4fdfd2d28dbb0ac856d828cb29f,
 7.0.203, ed1ae7cd88591ec52e1515c4a25d9a832eca29e8a0889549fea35a320e6e356e3806a17289f71fc0b04c36b006ae74446c53771d976c170fcbe5977ac7db1cb6,
+7.0.302, 9387bd804ed980ba1bc33093598ddbafa3a761e07d28916c94442cc329533d78a03bfc59d3066a1a861244302414e7e658b4e721b5bc825f623f8f908e748b7e,
```

Comment:
Since we are updating the stretch version, I think we also want to update the final stretch version in the constants.yaml to `7.0.302`

Reply:
True. Hopefully we can stop the support of Stretch versions overall in June.

Reply:
fingers crossed! Keeping these "final" constants up to date has been a pain haha. So if we don't, maybe we can have some actual logic that detects the most recent version in versionsToBuild and updating them for us

Reply:
Updated